site/pom.xml

```diff
@@ -79,9 +46,10 @@
                 </executions>
                 <configuration>
                     <executable>/bin/bash</executable>
-                    <workingDirectory>${project.build.outputDirectory}</workingDirectory>
+                    <workingDirectory>${project.basedir}</workingDirectory>
                     <arguments>
                         <argument>build-site.sh</argument>
+                        <argument>${project.build.outputDirectory}/public</argument>
```

Comment:
why not simply use `${project.build.outputDirectory}`. There is nothing else to be copied to that folder anyway, or is there?